src/send/mod.rs

```diff
@@ -3,10 +3,9 @@
 // file, you can obtain one at https://mozilla.org/MPL/2.0/.
 
 use rocket::{
-    data::{self, FromData}, http::Status, response::Failure, Data, Outcome, Request,
+    data::{self, FromData}, http::Status, response::Failure, Data, Outcome, Request
 };
 use rocket_contrib::{Json, Value};
-
```

Comment:
By convention we group `use` statements into `std`, external and internal, with a blank line separating each group. So this blank line is significant, let's put it back.